src/Symfony/Component/HttpKernel/Profiler/ProfilerStorageInterface.php

```diff
@@ -33,7 +33,7 @@ interface ProfilerStorageInterface
      * @param int|null $start The start date to search from
      * @param int|null $end   The end date to search to
      */
-    public function find(?string $ip, ?string $url, ?int $limit, ?string $method, int $start = null, int $end = null): array;
+    public function find(?string $ip, ?string $url, ?int $limit, string|array|null $method, int $start = null, int $end = null): array;
```

Comment:
BC break

Reply:
the interface is marked internal since +4years, and the profiler is generally a dev component
also https://symfony.com/doc/current/contributing/code/bc.html#using-our-interfaces

**of course we never know how devs use components in the wild**

but challenging a little bit here: perhaps interface and implementation class can be changed with an upgrade notice?
otherwise, defined a new method that takes a string with a specific separator that explode it, calls the old one if only one item or the new one as array

but this feature looks great and useful :)